src/sentry/rules/conditions/event_attribute.py

```diff
@@ -262,3 +258,38 @@ def passes(self, event: GroupEvent, state: EventState, **kwargs: Any) -> bool:
             return not attribute_values
 
         raise RuntimeError("Invalid Match")
+
+    def passes(self, event: GroupEvent, state: EventState, **kwargs: Any) -> bool:
+        attr = self.get_option("attribute")
+        try:
+            attribute_values = self._get_attribute_values(event, attr.lower())
```

Comment:
are we losing the logic for if `attr` is empty or None?

Reply:
I don't think its actually possible for `attr` to be none, it should have to be one of the choices? But if it is, i think it should still fail accordingly

Reply:
yeah but wouldn't it throw a `TypeError` or something? not a `KeyError` (should happen during the `attr.lower()` call

Reply:
that's only if it's `None` and not empty

Reply:
yeah i if its none then it'll throw an `AttributeError` which we don't catch, and if its empty then `_get_attribute_values` returns an empty list. But i think attr has to be in `ATTR_CHOICES` and i think some validation is probably done before this

Reply:
you can never really guarantee a frontend variable is sent successfully to the backend due to manual customer/hacker requests and just like faulty frontend stuff (like adblockers or js not properly loading). I'd either add a default of `''` to the `get_option` call or explicitly catch the exception that `.lower()` throws on a `None` type obejct.

without knowing too much about why the logic was there previously, in the old code we checked to see if it was empty/None and now we don't, which feels like a regression